packages/blockchain-wallet-v4-frontend/src/components/UnstoppableDomains/index.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { PureComponent } from 'react'
+import { connect, ConnectedProps } from 'react-redux'
+import { bindActionCreators, Dispatch } from 'redux'
+import styled from 'styled-components'
+
+import { BlockchainLoader, Text } from 'blockchain-info-components'
+import { ExtractSuccess } from 'core/types'
+import { actions } from 'data'
+import { RootState } from 'data/rootReducer'
+
+import { WalletFormType } from '../../../../../typings/redux-form'
```

Comment:
(minor) this should works

```
import { WalletFormType } from 'redux-form'
```

Reply:
Does it?

Reply:
sorry you are right it doesn't work per default, but you just need to export that type inside redux-form.d.ts 

```
...
export type WalletFormType
```